### DISCUSSION TITLE
Next.js strange inconsistent behaviour of `fallback` pages

### DISCUSSION THREAD

**Discussion by aleksey-shmatov** [vercel/vercel#4734] [Help]
Basically I have master page with list of items and detail pages for each of the items. I return only 5 item pages in `getStaticPaths` and set `fallback` to `true`. After deployment when I click on one of the items which was not pre-rendered usually nothing happens for a while and then suddenly pre-rendered page appears. I say usually because sometime I see fallback page with loading indicator first but this happens only sometimes. Am I supposed to handle this somehow on page with list of items? Can I enforce fallback page to be displayed?

**Comment by devklepacki** [comment]
+1 I also thought of any way to test fallback views, but wasn't able to figure this out by myself.
